Docs updates for Gene Expression

sidneymbell opened the pull request:
These should now match the paper. Pending @atarashansky 's review for accuracy.

sidneymbell commented on frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_3__Gene Expression Data Processing.mdx:
```diff
@@ -75,58 +75,31 @@ Only cells from sequencing assays that measure gene expression and don't require
 
 ## Gene-length Pre-normalization
 
-**WARNING** - The following process has not been implemented as the feature **currently only includes data that does not require gene-length normalization.**
```
I believe we now do this consistently - removing the warning. Lmk if this is not the case, @atarashansky .

sidneymbell commented on frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_3__Gene Expression Data Processing.mdx:
```diff
@@ -75,58 +75,31 @@ Only cells from sequencing assays that measure gene expression and don't require
 ## Removal of Noisy Ultra-low Expression Values
 
-After applying normalization, any gene/cell combination that had counts less or equal than 2 are set to missing data. This allows for removal of noise due to ultra-lowly expressed genes and provides a cleaner visualization.
+After applying normalization, any gene/cell combination counts less or equal than 2 are set to missing data. This allows for removal of noise due to ultra-lowly expressed genes and provides a cleaner visualization.
```
I need to update this line -- what cutoff value do we use here? Do we still apply this based on raw counts, before computing means? @atarashansky

atarashansky replied:
Change 2 to 3

atarashansky replied:
Otherwise, this is accurate

atarashansky commented on frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_3__Gene Expression Data Processing.mdx:
```diff
@@ -75,58 +75,31 @@ Only cells from sequencing assays that measure gene expression and don't require
 For each gene/cell type combination the following values are represented in the dot plot, either visually or in text upon hover.
 
-- Gene expression (dot color) – the average rankit-normalized gene expression among genes that have non-zero values.
+- Gene expression (dot color) – the average ln(CPM+1)-normalized gene expression among genes that have non-zero values.
```
it's not really CPM - it's sum-normalized to 10,000 counts prior to log transformation. don't really know what to call that

sidneymbell replied:
Changed

atarashansky commented on frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_5__Find Marker Genes.mdx:
```diff
@@ -29,10 +29,13 @@ For a selected cell type in a tissue,
-- For each gene, calculate the 5th percentile of the effect sizes across all comparisons. This is the marker score.
-  - Let $T_t$ be the set of cell types $\{c_1,\ldots,c_n\}$ in tissue $t$. The marker score ($S_{c_ig_j}$) for gene $g_j$ in cell type $c_i$ is $S_{c_ig_j}=p_5(\{e_{c_kg_j} : c_k \in T_t, c_k \neq c_i\})$ where $p_5$ is the 5th percentile operator and $e_{c_kg_j}$ is the effect size of the t-test comparing $g_j$ in $c_i$ to $c_k$.
+- For each gene, calculate the 10th percentile of the effect sizes across all comparisons. This is the marker score.
+  - Let $T_t$ be the set of cell types $\{c_1,\ldots,c_n\}$ in tissue $t$. The marker score ($S_{c_ig_j}$) for gene $g_j$ in cell type $c_i$ is $S_{c_ig_j}=p_10(\{e_{c_kg_j} : c_k \in T_t, c_k \neq c_i\})$ where $p_10$ is the 10th percentile operator and $e_{c_kg_j}$ is the effect size of the t-test comparing $g_j$ in $c_i$ to $c_k$.
```
$p_{10}$ - need brackets around subscripts > 1 length

atarashansky commented on frontend/doc-site/04__Analyze Public Data/4_2__Gene Expression Documentation/4_2_5__Find Marker Genes.mdx:
```diff
@@ -29,10 +29,13 @@ For a selected cell type in a tissue,
-  - Let $T_t$ be the set of cell types $\{c_1,\ldots,c_n\}$ in tissue $t$. The marker score ($S_{c_ig_j}$) for gene $g_j$ in cell type $c_i$ is $S_{c_ig_j}=p_5(\{e_{c_kg_j} : c_k \in T_t, c_k \neq c_i\})$ where $p_5$ is the 5th percentile operator and $e_{c_kg_j}$ is the effect size of the t-test comparing $g_j$ in $c_i$ to $c_k$.
+- For each gene, calculate the 10th percentile of the effect sizes across all comparisons. This is the marker score.
+  - Let $T_t$ be the set of cell types $\{c_1,\ldots,c_n\}$ in tissue $t$. The marker score ($S_{c_ig_j}$) for gene $g_j$ in cell type $c_i$ is $S_{c_ig_j}=p_10(\{e_{c_kg_j} : c_k \in T_t, c_k \neq c_i\})$ where $p_10$ is the 10th percentile operator and $e_{c_kg_j}$ is the effect size of the t-test comparing $g_j$ in $c_i$ to $c_k$.
+- For a few of comparisons, the 10th percentile can be a noisy metric. To improve its robustness, we bootstrap the distribution of effect sizes, taking the 10th percentile of each replicate, and averaging all replicates to get the final effect size.
```
For some comparisons*

atarashansky replied:
bootstrap the distribution of effect sizes for each gene* maybe is more clear?

sidneymbell commented:
Incorporated all comments from Alec. Also updated notification and docs to read `CPTT` (counts per ten thousand) rather than `CPM` (counts per million).
